packages/perseus/src/widgets/radio/widget.jsx

```diff
@@ -135,17 +147,24 @@ class Radio extends React.Component<Props> {
     //
     // NOTE(mdr): This method expects to be auto-bound. If this component is
     //     converted to an ES6 class, take care to auto-bind this method!
-    updateChoices: ($FlowFixMe) => void = (newValueLists) => {
+    updateChoices: (
+        newValueLists: $ReadOnly<{
+            checked: $ReadOnlyArray<boolean>,
+            crossedOut: $ReadOnlyArray<boolean>,
+        }>,
+    ) => void = (newValueLists) => {
         const {choiceStates, choices} = this.props;
 
         // Construct the baseline `choiceStates` objects. If this is the user's
         // first interaction with the widget, we'll need to initialize them to
         // new objects with all fields set to the default values. Otherwise, we
         // should clone the old `choiceStates` objects, in preparation to
         // mutate them.
-        let newChoiceStates;
+        let newChoiceStates: $ReadOnlyArray<ChoiceState>;
         if (choiceStates) {
-            newChoiceStates = choiceStates.map((state) => ({...state}));
+            newChoiceStates = choiceStates.map((state: ChoiceState) => ({
+                ...state,
+            }));
         } else {
             newChoiceStates = choices.map(() => ({
```

Comment:
**style nit (non-blocking):** I try to avoid `let` unless absolutely necessary.  In this case you could use `const` by reformulating this as a ternary, e.g.
```
const newChoiceStates = choiceStates
   ? choiceStates.map((state: ChoiceState) => ({...state}))
   : choices.map(() => ({ ... });
```